hs_core/static/css/iThing.css

```diff
@@ -0,0 +1,251 @@
+/**
+ * Theme for jQRangeSlider
+ * Inspired by http://cssdeck.com/item/381/itunes-10-storage-bar
+ *        and http://cssdeck.com/item/276/pure-css-arrow-with-border-tooltip
+ */
+
+.ui-rangeSlider{
+  height: 30px;
+  padding-top: 40px;
+}
+
+.ui-rangeSlider,
+.ui-rangeSlider-container,
+.ui-rangeSlider-arrow{
+  -webkit-box-sizing:content-box;
+     -moz-box-sizing:content-box;
+          box-sizing:content-box;
```

Comment:
Please remove `box-shadow` and `box-sizing` prefixes. Both have plenty of wide support among browsers.